app/components/events-section.tsx

```diff
@@ -0,0 +1,50 @@
+export default function EventsSection() {
```

Comment:
In the future, the `EventsSection` may receive an array of recommended event objects and then render an `EventPreview` container for each event. Happy to merge it like this for now, but we could also already add a `events` prop and have the mock data live in the `routes/_index` file?